_posts/2016-09-17-solidity-0.4.2-release-announcement.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.4.2 Release Announcement'
+date: '2016-09-17'
+author: Solidity Team
+category: Releases
+---
+
```

Comment:
Maybe we can add an introductory one-liner here "This is a bugfix release fixing..."?